epi_scanner/viz.py

```diff
@@ -177,14 +228,170 @@ async def plot_pars_map_altair(
+):
+    map_rate = get_rate_map(
+        years, statemap, q.client.data_table, q.client.parameters
+    )
+    legend_table = pd.DataFrame(
+        [
+            [0, 1, 0, bins[0], "a"],
+            [1, 2, bins[0], bins[1], "b"],
+            [2, 3, bins[1], bins[2], "c"],
+            [3, 4, bins[2], bins[3], "d"],
+            [4, 5, bins[3], bins[3] + 1, "e"],
+        ],
+        columns=["x1", "x2", "v1", "v2", "color"],
+    )
+
+    map = alt.Chart(data=map_rate,).mark_geoshape(
+        fillOpacity=0.5, fill="grey", stroke="#000", strokeOpacity=0.5
+    ) + alt.Chart(data=map_rate,).mark_geoshape(
+        stroke="#000", strokeOpacity=0.5
+    ).encode(
+        color=alt.Color(
+            f"{column}:Q",
+            sort="ascending",
+            scale=alt.Scale(type="threshold", domain=bins, range=color_list),
+            legend=None,
+        ),
+        tooltip=["name_muni", column + ":Q"],
+    ).properties(
+        title={
+            "text": f"{title} {years[0]}",
+            "fontSize": 15,
+            "anchor": "start",
+        },
+        width=500,
+        height=400,
+    )
+
+    legend = (
+        alt.Chart(legend_table, height=70)
+        .mark_rect()
+        .encode(
+            x=alt.X("x1", scale=alt.Scale(), axis=None),
+            x2="x2",
+            y=datum(0, scale=alt.Scale(), axis=None),
+            y2=datum(1),
+            color=alt.Color(
+                "color", legend=None, scale=alt.Scale(range=color_list)
+            ),
+        )
+        + alt.Chart(legend_table)
+        .mark_text(size=10)
+        .encode(x="x1", text="v1", y=datum(-0.8))
+        + alt.Chart(legend_table)
+        .mark_rule(opacity=0.6)
+        .encode(x="x1", y=datum(1), y2=datum(-0.4))
+        + alt.Chart()
+        .mark_rule()
+        .encode(x=datum(0), x2=datum(1.95), y=datum(1.1))
+        + alt.Chart()
+        .mark_rule()
+        .encode(x=datum(2.05), x2=datum(2.95), y=datum(1.1))
+        + alt.Chart()
+        .mark_rule()
+        .encode(x=datum(3.05), x2=datum(5), y=datum(1.1))
+        + alt.Chart()
+        .mark_text(size=10)
+        .encode(x=datum(1), y=datum(1.5), text=datum("Underestimated"))
+        + alt.Chart()
+        .mark_text(size=10)
+        .encode(x=datum(2.5), y=datum(1.5), text=datum("Good"))
+        + alt.Chart()
+        .mark_text(size=10)
+        .encode(x=datum(4), y=datum(1.5), text=datum("Overestimated"))
+        + alt.Chart()
+        .mark_text(size=12, fontWeight="bold")
+        .encode(
+            x=datum(2.5),
+            y=datum(2.5),
+            text=datum("Model Evaluation"),
+        )
+        + alt.Chart()
+        .mark_text(size=11, fontWeight="bold")
+        .encode(
+            x=datum(2.5),
+            y=datum(-1.6),
+            text=datum("Observed Cases/Estimated Cases"),
+        )
+    ) + alt.Chart().mark_text(
+        text="* Cities in gray didn't\nhave an epidemic.",
```

Comment:
change to: "* Cities in gray did not have an epidemic detected."

Reply:
@CarCesar mesmo após sua mudança ainda não está renderizando de forma legível a legenda, acho que talvez você terá que alterar alguma configuração do altair, verifique se não tem algum parâmetro que altera a resolução. Para você localmente aparece legível? 

Para mim aparece assim: 
![Captura de Tela 2024-12-03 às 16 22 00](https://github.com/user-attachments/assets/39512f80-fc09-4cae-b751-41f5da148913)

Reply:
Ok, no meu está um pouco mais legível, mas também não está perfeito. Ontem, comecei a pesquisar sobre e não entrei nenhuma forma de aumentar o espaçamento entre as letras. Uma alternativa que poderia ajudar seria a utilização de fonte `monospace`, mas ficaria uma fonte diferente da fonte do resto do servidor, acho que não seria o ideal. 
Vou pesquisar um pouco mais.